src/CqlBuilderPanel/codesSection/codesSubSection/codeSubSection/ResultsSection.tsx

```diff
@@ -205,6 +211,8 @@ export default function ResultsSection({
                         <td key={cell.id} tw="p-2">
                           {cell.column.id === "status"
                             ? getCodeStatus(cell.getValue())
+                            : cell.column.id === "fhirVersion"
+                            ? getFhirCodeSystemVersion(cell.getValue())
```

Comment:
same comment as above regarding model checks

Reply:
maybe something like this, where the model specific cell component is created in a parent directory with model specific path (like we do in madie-measure for all the model-specific components)

```
  const ResultCellComponent = useMemo(
    () =>
      lazy(() => {
        if (measureModel?.includes("QDM")) {
          return import("../../qdm/ResultCell");
        }
        if (measureModel?.includes("QI-Core")) {
          return import("../../qicore/ResultCell");
        } else {
          return import("../../Empty");
        }
      }),
    [measureModel]
  );
  ```
  
  ```
  <td key={cell.id} tw="p-2">
      {cell.column.id === "status" ? (
        getCodeStatus(cell.getValue())
      ) : (
        <ResultCellComponent cell={cell} />
      )}
</td>
```